CHANGES.txt

```diff
@@ -8,6 +8,11 @@ Unreleased
 - SQLAlchemy DDL: Allow turning off column store using ``crate_columnstore=False``.
   Thanks, @fetzerms.
 
+- SQLAlchemy DDL: Allow setting ``server_default`` on columns to enable
+  server-generated defaults.
+
+- Allow handling "aware" datetime values when inserting or updating.
```

Comment:
```suggestion
- Allow handling datetime values with time zone info when inserting or updating.
```

Reply:
Suggestion: maybe we could also an example?

Reply:
I've added a corresponding example to the documentation at `client.rst`. It is not optimal, but the best thing we can do to show the features, and not destroy the storyline of that documentation section too much.